scipy/stats/tests/test_continuous_basic.py

```diff
@@ -542,7 +545,8 @@ def check_pdf_logpdf_at_endpoints(distfn, args, msg):
             "divide by zero encountered in power",  # gengamma
             "invalid value encountered in add",  # genextreme
             "invalid value encountered in subtract",  # gengamma
-            "invalid value encountered in multiply"  # recipinvgauss
+            "invalid value encountered in multiply",  # recipinvgauss
+            "invalid value encountered in log"  # studentized_range
```

Comment:
Does this still happen even though you took terms out of the log?

Reply:
@DominicChm?

Reply:
I don't remember what platform this took place on, but I don't see that error locally. I'll take it off along with the next set of commits and we can see.